Podverse/FiltersTableHeaderView.swift

```diff
@@ -0,0 +1,277 @@
+                                                  constant: 0)
+        
+        let bottomBorderBottom = NSLayoutConstraint(item: self,
+                                              attribute: .bottom,
+                                              relatedBy: .equal,
+                                              toItem: self.bottomBorder,
+                                              attribute: .bottom,
+                                              multiplier: 1,
+                                              constant: 0)
+        
+        let bottomBorderTrailing = NSLayoutConstraint(item: self.bottomBorder,
+                                                   attribute: .trailing,
+                                                   relatedBy: .equal,
+                                                   toItem: self,
+                                                   attribute: .trailing,
+                                                   multiplier: 1,
+                                                   constant: 0)
+        
+        let bottomBorderHeight = NSLayoutConstraint(item: self.bottomBorder,
+                                                 attribute: .height,
+                                                 relatedBy: .equal,
+                                                 toItem: nil,
+                                                 attribute: .notAnAttribute,
+                                                 multiplier: 1,
+                                                 constant: 1)
+        
+        self.addSubview(self.bottomBorder)
+        
+        self.addConstraints([filterLeading, filterTop, filterHeight, filterWidth, sortingLeading, sortingTop, sortingHeight, sortingWidth, topBorderLeading, topBorderTop, topBorderTrailing, topBorderHeight, bottomBorderLeading, bottomBorderBottom, bottomBorderTrailing, bottomBorderHeight])
+        
+    }
+    
+    func showSortByTimeRangeMenu(vc: Any) {
+        
+        let alert = UIAlertController(title: "Time Range", message: nil, preferredStyle: .actionSheet)
+        
+        alert.addAction(UIAlertAction(title: "Day", style: .default, handler: { action in
+            self.sortByTopDay()
+        }))
+        
+        alert.addAction(UIAlertAction(title: "Week", style: .default, handler: { action in
+            self.sortByTopWeek()
+        }))
+        
+        alert.addAction(UIAlertAction(title: "Month", style: .default, handler: { action in
+            self.sortByTopMonth()
+        }))
+        
+        alert.addAction(UIAlertAction(title: "Year", style: .default, handler: { action in
+            self.sortByTopYear()
+        }))
+        
+        alert.addAction(UIAlertAction(title: "All Time", style: .default, handler: { action in
+            self.sortByTopAllTime()
+        }))
+        
+        alert.addAction(UIAlertAction(title: "Cancel", style: .cancel, handler: nil))
+        
+        if let vc = vc as? PVViewController {
```

Comment:
Is there a reason to specifically check for PVViewController instead of Generally UIViewController?

Reply:
I want it to work with PVViewController as well as UIViewController. Since PVViewController is a subclass of UIViewController, can I just use UIViewController here?

Reply:
Yes exactly. PVViewcontroller is subclassing UIViewcontroller so it is a UIViewcontroller so it should work only checking for UIViewController :)

Reply:
k done